lms/static/scripts/frontend_apps/utils/test/google-picker-client-test.js

```diff
@@ -113,8 +113,12 @@ describe('GooglePickerClient', () => {
     });
   });
 
+  afterEach(() => {
+    $imports.$restore();
```

Comment:
Good spot. A flaw in the current `$imports` API is that this mistake is easy to make.